src/app/core/entity/entity-mapper/entity-mapper.service.ts

```diff
@@ -195,7 +195,8 @@ export class EntityMapperService {
   }
 
   protected setEntityMetadata(entity: Entity) {
-    const newMetadata = new UpdateMetadata(this.currentUser.value?.name);
+    // TODO use Entity (CurrentlyLoggedIn) or keep AuthUser?
+    const newMetadata = new UpdateMetadata(this.sessionInfo.value?.name);
```

Comment:
really interesting question ...
AuthUser feels a bit more like secured accountability. But for display this in the UI in future, the UserEntity would be much easier to use. So maybe rather optimize for that?